code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -221,80 +215,62 @@
 // *************************************
 
 
-/obj/item/reagent_containers/glass/bottle/nutrient
-	name = "bottle of nutrient"
+/obj/item/reagent_containers/glass/jug/nutrient
+	name = "jug of nutrient"
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle16"
-	volume = 50
+	icon_state = "plastic_jug"
 	w_class = WEIGHT_CLASS_TINY
-	amount_per_transfer_from_this = 10
-	possible_transfer_amounts = list(1,2,5,10,15,25,50)
 
-/obj/item/reagent_containers/glass/bottle/nutrient/New()
+/obj/item/reagent_containers/glass/jug/nutrient/Initialize()
```

Comment:
`Initialize` takes `mapload` as default parameter. It's best to also include it so others won't get confused later on. Same for the others in here